signer/src/bitcoin/utxo.rs

```diff
@@ -2783,136 +2831,6 @@ mod tests {
-    #[test_case(vec![
-        create_deposit(10_000, 10_000, 0),
-    ], 0, 0, 0, 0; "should_handle_empty_deposit_list")]
-    #[test_case(vec![
-        create_deposit(10_000, 0, 0),
-        create_deposit(11_000, 10_000, 0),
-        create_deposit(9_000, 10_000, 0),
-    ], 1, 9_000, 10_000, 10_000; "should_skip_invalid_fee_and_accept_valid_deposits")]
-    #[test_case(vec![
-        create_deposit(10_001, 10_000, 0),
-    ], 0, 0, 10_001, 10_000; "should_reject_single_deposit_exceeding_max_mintable")]
-    #[test_case(vec![
-        create_deposit(10_000, 10_000, 0),
-    ], 0, 0, 8_000, 10_000; "should_reject_single_deposit_exceeding_per_deposit_cap")]
-    #[tokio::test]
-    async fn test_construct_transactions_filters_deposits_over_max_mintable(
-        deposits: Vec<DepositRequest>,
-        num_accepted_requests: usize,
-        accepted_amount: u64,
-        per_deposit_cap: u64,
-        max_mintable: u64,
-    ) {
-        // Each deposit and withdrawal has a max fee greater than the current market fee rate
-        let public_key = XOnlyPublicKey::from_str(X_ONLY_PUBLIC_KEY1).unwrap();
-        let requests = SbtcRequests {
-            deposits: deposits,
-            withdrawals: vec![],
-            signer_state: SignerBtcState {
-                utxo: SignerUtxo {
-                    outpoint: generate_outpoint(300_000, 0),
-                    amount: 300_000_000,
-                    public_key,
-                },
-                fee_rate: 25.0,
-                public_key,
-                last_fees: None,
-                magic_bytes: [0; 2],
-            },
-            num_signers: 10,
-            accept_threshold: 8,
-            sbtc_limits: SbtcLimits::new(
-                None,
-                Some(Amount::from_sat(per_deposit_cap)),
-                None,
-                Some(Amount::from_sat(max_mintable)),
-            ),
-        };
-        let txs = requests.construct_transactions().unwrap();
-        let nr_requests = txs.iter().map(|tx| tx.requests.len()).sum::<usize>();
-        let total_amount: u64 = txs
-            .iter()
-            .map(|tx| {
-                tx.requests
-                    .iter()
-                    .map(|req| req.as_deposit().unwrap().amount)
-            })
-            .flatten()
-            .sum();
-        assert_eq!(nr_requests, num_accepted_requests);
-        assert_eq!(total_amount, accepted_amount);
-    }
-
-    #[test_case(
-        create_deposit(
-            DEPOSIT_DUST_LIMIT + SOLO_DEPOSIT_TX_VSIZE as u64,
-            10_000,
-            0
-        ),
-        true; "deposit amounts over the dust limit accepted")]
-    #[test_case(
-        create_deposit(
-            DEPOSIT_DUST_LIMIT + SOLO_DEPOSIT_TX_VSIZE as u64 - 1,
-            10_000,
-            0
-        ),
-        false; "deposit amounts under the dust limit rejected")]
-    fn deposit_requests_respect_dust_limits(req: DepositRequest, is_included: bool) {
```

Comment:
Why delete these?

Reply:
I moved them in the filter_deposits test

https://github.com/stacks-network/sbtc/pull/1124/files#diff-aef027d6a4d764f2511fb7b7065d4b2b7ecff26b7ff1e59deaa25955705f1736R2908-R2914
and 
https://github.com/stacks-network/sbtc/pull/1124/files#diff-aef027d6a4d764f2511fb7b7065d4b2b7ecff26b7ff1e59deaa25955705f1736R2983

Reply:
I see, well I would keep them both. Also, it looks like you missed the "deposit amounts over the dust limit accepted" one right?